src/infohud.asm

```diff
@@ -612,7 +612,12 @@ ih_update_hud_code:
     ; Map visible, so draw map counter over item%
     LDA !sram_top_display_mode : CMP !TOP_DISPLAY_VANILLA : BEQ .mmVanilla
     LDA !ram_map_counter : LDX #$0014 : JSR Draw3
-    LDA !sram_display_mode : CMP #!IH_MODE_SHINETUNE_INDEX : BNE .mmRoomTimer
+
+    ; Skip room timer when shinetune or walljump enabled
+    LDA !sram_display_mode : CMP #!IH_MODE_SHINETUNE_INDEX : BEQ .mmSkipRoomTimer
+    CMP #!IH_MODE_WALLJUMP_INDEX : BNE .mmRoomTimer
```

Comment:
Do you think this is worth having a flag that the HUD mode sets and this routine clears?